enterprise/coderd/coderd.go

```diff
@@ -943,6 +952,18 @@ func (api *API) CheckBuildUsage(ctx context.Context, store database.Store, templ
 	// When there are no licenses installed, a noop usage checker is used
 	// instead.
```

Comment:
Dude to this comment above (and the entitlements code in the function above), external agents won't be blocked if there are no licenses installed (e.g. an enterprise build in free mode). Is that what you want? If it's not, I can walk you through a few changes to fix this.